integration_test/scripts/run-store-scp-and-wait.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
-# run_step scp_port scp_finish_port
+
+set -e
+set -o pipefail
+
+export PATH=/opt/apache-maven-3.6.3/bin:$PATH
 cd /workspace/dcm4che/dcm4che-tool/dcm4che-tool-storescp
 mvn exec:java -ntp -Dexec.mainClass=org.dcm4che3.tool.storescp.StoreSCP -Dexec.args='--bind STORESCP@'$1':'$2' --directory /workspace/integration_test/storescp-data --accept-unknown' &
 apt-get -qq update
 apt-get -qq install -y netcat-openbsd
 echo 'Waiting for connection on port '$3' to finish store-scp step'
-nc -l -p $3
+nc -l -p $3 &
```

Comment:
This line could be removed. We don't even pass in the third parameter to this script, so this isn't doing anything. 
Removing is fine because we wait for store scp later in the script